src/main/java/bio/terra/service/filedata/SynapseDataResultModel.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.service.filedata;
```

Comment:
Sorry, I'm not really getting why we need the synapse and BW specific classes. Seems like the responses should be the same no?

Reply:
The difference is the row result data type. BigQuery maps into Map<String, Object>, while the Synapse results map into Map<String, Optional<Object>>. I tried switching both to the same return type, but dealing with the optional was quite a pain in either direction (both getting rid of the optional and keeping it for both), so I ended up just splitting out the data model.